engine/src/main/java/org/pentaho/di/trans/step/errorhandling/AbstractFileErrorHandler.java

```diff
@@ -40,6 +40,8 @@
 import org.pentaho.di.i18n.BaseMessages;
 import org.pentaho.di.trans.step.BaseStep;
 
+import static org.apache.commons.lang.StringUtils.isBlank;
```

Comment:
We don't import static methods other than in unit tests.

Reply:
Is that standard though? @kurtwalker fixed some of my code adding a similar method to ensure that null and empty strings are covered.  I applied the same logic here.  How else would you like me to do this besides manually writing: "if encoding != "" && encoding != null"

Reference: https://github.com/pentaho/pentaho-vertica-bulkloader/pull/76/commits/d0786be799a2fb4bed5126ea4fd37e99707bf337

Reply:
There was a time that checkstyle didn't allow it.  I objected and got the dev team at the time to agree.  I use static imports frequently in production code, as does at least all the devs on ackbar.